src/templates/datagrid.latte

```diff
@@ -14,7 +14,7 @@
  * @param InlineEdit|null $inlineAdd   Inline add data
  *}
 
-<div class="datagrid datagrid-{$control->getFullName()}" data-refresh-state="{link refreshState!}">
+<div class="{block grid-class}datagrid datagrid-{$control->getFullName()}{/block}" data-refresh-state="{link refreshState!}">
```

Comment:
Maybe pls `{block datagrid-class}`? (There probably more references to a "datagrid" than to a "grid"). Thanks

Reply:
Sure, everything is called datagrid... changed that